x-pack/legacy/plugins/actions/server/lib/execute.ts

```diff
@@ -37,9 +37,43 @@ export async function execute({
   };
   const validatedConfig = validateActionTypeConfig(actionType, mergedActionTypeConfig);
   const validatedParams = validateActionTypeParams(actionType, params);
-  await actionType.executor({
-    services,
-    config: validatedConfig,
-    params: validatedParams,
-  });
+
+  let result;
+  let error;
+  try {
+    result = await actionType.executor({
+      services,
+      config: validatedConfig,
+      params: validatedParams,
+    });
+  } catch (err) {
+    error = err;
+  }
+
+  const { actionTypeId, description } = action.attributes;
+  const actionLabel = `${actionId} - ${actionTypeId} - ${description}`;
+
+  if (error != null) {
+    services.log(
+      ['info', 'x-pack', 'actions'],
```

Comment:
info seems like the wrong log level for an error.

Reply:
Ah, yes.  And for the `info` one below (success), wondering if that should be a `debug`.  How chatty is the Kibana log?  Seems usually quiet to me.

Reply:
Yeah debug is probably better for the one below.